src/buildercore/terraform.py

```diff
@@ -401,6 +408,7 @@ def init(stackname, context):
                 'fastly': {
                     # exact version constraint
                     'version': "= %s" % PROVIDER_FASTLY_VERSION,
+                    'api_key': "${data.%s.%s.data[\"api_key\"]}" % (DATA_TYPE_VAULT_GENERIC_SECRET, DATA_NAME_VAULT_FASTLY_API_KEY),
```

Comment:
is this jsonp?

Reply:
Terraform "Interpolation Syntax" https://www.terraform.io/docs/configuration/interpolation.html#attributes-of-a-data-source, the underlying data structure is [HCL](https://www.terraform.io/docs/configuration/syntax.html) which is JSON written with a different syntax.

Reply:
ah, I should have guessed. That was the only drawback to using Terraform for me, learning this weird custom json compatible format.

Reply:
TBF: That is a common complaint about the json ... there are people I know who have written tools around transforming other more clearer formats to this format.